src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -187,7 +188,7 @@ bool MulticopterLandDetector::_get_landed_state()
 		// falling consider it to be landed. This should even sustain
 		// quite acrobatic flight.
 		if ((_min_trust_start > 0) &&
-		    (hrt_elapsed_time(&_min_trust_start) > 8 * 1000 * 1000)) {
+		    (hrt_elapsed_time(&_min_trust_start) > 8e6)) {
```

Comment:
I just realized that 8e6 is actually a double, so you are changing the type here (and the other places as well).